test-data/unit/check-modules.test

```diff
@@ -1175,6 +1175,33 @@ reveal_type(x)
 main:1: error: Incompatible types in assignment (expression has type "int", variable has type "str")
 main:2: error: Revealed type is 'builtins.str'
 
+[case testSymmetricImportCycle1]
+import a
+[file a.py]
+import b
+def f() -> int:
+    return b.x
+y = 0 + 0
+[file b.py]
+import a
+def g() -> int:
+    return a.y
```

Comment:
Reveal the type of `a.y` (to verify that it doesn't somehow fall back to `Any`).